src/tests/ut/visit/status_test.cpp

```diff
@@ -0,0 +1,23 @@
+#include "gtest/gtest.h"
+
+#include "status.hpp"
+
+TEST(EnumStatusTest1, ToStringShouldReturnCanceled)
+{
+    EXPECT_EQ(toString(Status::Canceled), "Canceled");
+}
+
+TEST(EnumStatusTest2, ToStringShouldReturnConfirmed)
+{
+    EXPECT_EQ(toString(Status::Confirmed), "Confirmed");
+}
+
+TEST(EnumStatusTest3, ToStringShouldReturnRelocated)
+{
+    EXPECT_EQ(toString(Status::Relocated), "Relocated");
+}
+
+TEST(EnumStatusTest4, ToStringShouldReturnRelocated)
+{
+    EXPECT_EQ(toString(Status::Visited), "Visited");
+}
```

Comment:
could be parametrized to make sure if we check each case or just single test checking each case
Naming with "1", "2", "3", "4" isn't also pretty, so paramterized test would let you get rid of that

Reply:
Done